README.md

```diff
@@ -39,6 +40,19 @@ from happening. Most logging happens at `INFO`, which is the default setting.
 
   - If you are on WSL, installing docker on the linux system wouldn't work as linux itself is running in a container. Install docker desktop instead following these instructions: https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers
 
+- To access AWS Bedrock models and other relevant AWS permissions from a local aiproxy container, copy the `aws_access_key_id`,
+  `aws_secret_access_key`, and `region` from 'AIProxy Local IAM User' in the Engineering 1Password
+  folder and add them to `.aws/credentials` under the name `aiproxy_development`.
+
+  Example:
+
+  ```
+  [bedrock_development]
```

Comment:
apologies, as soon as I commented I saw that you had updated this to aiproxy_development in some places, but it looks like the one place I looked at had been missed.